app/loading.tsx

```diff
@@ -0,0 +1,5 @@
+const Loading = () => {
+  return <div>Loading...</div>;
+};
+
+export default Loading;
```

Comment:
Can you create a spinner that would be fixed, it will be centered vertically and horizontally.
So we would have the spinner, and <p> Loading... </p>under it.
The spinner should have a default color, but we should be able to change the color.
For the look I am thinking something like this, but if you find a better looking option feel free to implement it.
![1_6Pv-vKJnzLZNn1iR1GmFtg](https://github.com/learnhubmkd/app/assets/105276163/7ebef608-85a5-49b6-a4c5-6b4160cd25fd)